src/pages/Main/Direction/Direction.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styles from '../MainPage.module.scss';
+import icon001 from '../../../assets/icons/icon_001.png';
+import icon002 from '../../../assets/icons/icon_002.png';
+import icon003 from '../../../assets/icons/icon_003.png';
+import icon004 from '../../../assets/icons/icon_004.png';
+import icon005 from '../../../assets/icons/icon_005.png';
+
+interface DirectionItemProps {
```

Comment:
The same problem with naming from this [PR](https://github.com/Train-lab-intern/frontend_dev/pull/47#pullrequestreview-2110518673) (check all comments, please, for better understanding).

Reply:
i agree, fix